app/views/layouts/application.html.slim

```diff
@@ -100,8 +100,15 @@ html
     / boxes. See that JavaScript file for more details.
     .js-shared-modal
       .md-container.container
-        .col.col-10.col-md-8.col-sm-10.col-xs-12.col-center.sub-panel.sub-panel--panel--modal
-          .md-content
-          = link_to "#", class: "md-close js-deny"
-            = render "application/icon_x"
+        - if content_for?(:head_style_tags) || params.has_key?(:bootstrap4)
```

Comment:
technically the check against `content_for?(:head_style_tags)` isn't correct, as it could have other content besides the bootstrap 4 css document.

Reply:
This is a temporary switch because we need alternate modal markup for Bootstrap 4. Once that transition is complete, this check can be removed.

Reply:
@kpfefferle could you add a FIXME comment there?